build-system/test-configs/dep-check-config.js

```diff
@@ -440,19 +440,19 @@ exports.rules = [
   // Doubleclick.js will be deleted from the repository at that time.
   // Please see https://github.com/ampproject/amphtml/issues/11834
   // for more information.
-  // Do not add any additional files to this whitelist without express
+  // Do not add any additional files to this allowlist without express
   // permission from @bradfrizzell, @keithwrightbos, or @robhazan.
   {
     mustNotDependOn: ['ads/google/doubleclick.js'],
-    whitelist: [
-      /** DO NOT ADD TO WHITELIST */
+    allowlist: [
+      /** DO NOT ADD TO allowlist */
       'ads/ix.js->ads/google/doubleclick.js',
       'ads/imonomy.js->ads/google/doubleclick.js',
       'ads/navegg.js->ads/google/doubleclick.js',
-      /** DO NOT ADD TO WHITELIST */
+      /** DO NOT ADD TO allowlist */
       'ads/openx.js->ads/google/doubleclick.js',
       'ads/pulsepoint.js->ads/google/doubleclick.js',
-      /** DO NOT ADD TO WHITELIST */
+      /** DO NOT ADD TO allowlist */
```

Comment:
These three should probably be all caps.

Reply:
ah darn i missed this. tiny followup incoming!